config/environments/development.rb

```diff
@@ -2,7 +2,7 @@
 
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-config.webpacker.check_yarn_integrity = true
+config.webpacker.check_yarn_integrity = false
```

Comment:
I switched this to false since I've been jumping back and forth between docker and local and this was blocking me on local since I didn't want to download all of the npm dependencies.